src/helpers/readme-helpers-trait.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * The readme.txt helpers trait file.
+ *
+ * @since   1.1.0
+ *
+ * @package Theme_Sniffer\Helpers
+ */
+
+declare( strict_types=1 );
+
+namespace Theme_Sniffer\Helpers;
+
+/**
+ * Sniffer helpers trait
+ *
+ * This trait contains some helper methods.
+ *
+ * @since 1.1.0
+ */
+trait Readme_Helpers {
+
+	/**
+	 * License data from imported from json.
+	 *
+	 * @var array $license_data
+	 *
+	 * @since 1.1.0
+	 */
+	protected $license_data;
+
+	/**
+	 * Initialize class and set class properties.
+	 *
+	 * @since 1.1.0
+	 */
+	public function init() {
+		$this->license_data = $this->set_license_data();
+	}
+
+	/**
+	 * Gets license validation data.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @return array License validation data.
+	 */
+	public function set_license_data() : array {
+		$file = WP_PLUGIN_DIR . plugin_dir_path( '/theme-sniffer/assets/build/licenses.json' );
```

Comment:
If the `licenses.json` can be pulled from somewhere we should pull this then from the vendors of where ever we pull it from.